fossevents/events/services.py

```diff
@@ -59,10 +88,42 @@ def send_confirmation_mail(event, is_update=False):
     :param is_update boolean
     """
 
-    # get all end users both candidates and employers
-    moderators_email = User.objects.filter(is_staff=True).values_list('email', flat=True)
+    # get all moderator's email
+    moderators_email = User.objects.filter(Q(is_staff=True) | Q(is_moderator=True) |
+                                           Q(is_superuser=True)).distinct().values_list('email', flat=True)
```

Comment:
same goes here, can be added as separate function to fetch moderators email to re-use at other places.